### PR TITLE
[zh] sync and add anchors to custom-resource-definition-versioning.md

### PR DESCRIPTION
**Pull request by chengxiangdong**
- sync with en
- add anchors
```
content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md
```

### CODE REVIEW

File: content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md [hunk 1]
```diff
@@ -629,7 +629,7 @@ other than `labels` and `annotations`.
 Attempted changes to `name`, `UID` and `namespace` are rejected and fail the request
 which caused the conversion. All other changes are ignored. 
 -->
-#### 被允许的变更
+#### 被允许的变更 {#permissible-mutations}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:632]: Normally, providing an anchor to heading 2 and 3 is a good practice.

Mostly, it is not necessary to add anchors to heading 4. Do you find any link points to this heading?

Comment (chengxiangdong) [reply content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md]: yeah, I get it, thanks.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**